frontend/src/components/Events/EventDetails.js

```diff
@@ -28,17 +31,27 @@ const Button1 = styled(Button)({
 })
 
 function EventDetails(props) {
+
+    const addParticipant = () => {
+        if (!JSON.stringify(props.event.participants).includes(JSON.stringify({uid: firebase.auth().currentUser.uid, email:firebase.auth().currentUser.email}))) {
+            axios.patch('http://localhost:3001/events/participant', {_id: props.event._id, participant: {uid: firebase.auth().currentUser.uid, email:firebase.auth().currentUser.email}}).then(() => {
```

Comment:
The URL should exist inside an environment variable rather than being hard coded in. If you go inside the `.env.local` file we used for setting up firebase, you could put a line like `REACT_APP_SERVER_URL="'http://localhost:3001/"`. Then, you could create an axios instance using 
```const axios = require('axios').create({
  baseURL: REACT_APP_SERVER_URL
});```
and then finally